src/cryptography/hazmat/primitives/asymmetric/ec.py

```diff
@@ -252,6 +264,28 @@ def __init__(self, algorithm):
     algorithm = utils.read_only_property("_algorithm")
 
 
+@utils.register_interface(KeyDerivationFunction)
+class ECDHnoKDF(object):
```

Comment:
Having spoken with @alex I think it might actually be better to just leave this functionality out entirely until we have a clear use-case for it. How do you feel about that @simo5 ? Are you aware of any cases where you really need access to the raw ECDH output?